### ISSUE TITLE
adjusting learning rate

### ISSUE THREAD

**Issue by mangdian** [alterzero/RBPN-PyTorch#31]
when training the network using 9 frames or 11 frames, is it necessary to adjust the learning rate?

**Comment by alterzero** [comment]
I believe training on 9 frames or more will be harder.
When I trained 11 frames, it is fine-tuned from F7's model.
So, you can start from fine-tuning model with lower learning rate.

**Comment by mangdian** [comment]
how many frames did you use to get the best experimental results?
